app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -2,25 +2,58 @@ package org.simple.clinic.bp.history
 
 import android.content.Context
 import android.util.AttributeSet
+import android.view.LayoutInflater
 import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.recyclerview.widget.LinearLayoutManager
+import kotlinx.android.synthetic.main.screen_bp_history.view.*
+import org.simple.clinic.R
 import org.simple.clinic.bp.BloodPressureMeasurement
+import org.simple.clinic.bp.history.adapter.BloodPressureHistoryListItem
+import org.simple.clinic.bp.history.adapter.BloodPressureHistoryListItemDiffCallback
 import org.simple.clinic.main.TheActivity
+import org.simple.clinic.summary.PatientSummaryConfig
+import org.simple.clinic.util.UtcClock
+import org.simple.clinic.widgets.ItemAdapter
 import java.util.UUID
+import javax.inject.Inject
 
 class BloodPressureHistoryScreen(
     context: Context,
     attrs: AttributeSet
 ) : ConstraintLayout(context, attrs), BloodPressureHistoryScreenUi, BloodPressureHistoryScreenUiActions {
 
+  @Inject
+  lateinit var utcClock: UtcClock
+
+  @Inject
+  lateinit var config: PatientSummaryConfig
+
+  private val bloodPressureHistoryAdapter = ItemAdapter(BloodPressureHistoryListItemDiffCallback())
+
+  init {
+    LayoutInflater.from(context).inflate(R.layout.screen_bp_history, this, true)
```

Comment:
This should not be necessary since `screenRouter` will automatically inflate the view.